# GenericComponentMsg.msg

uint8 part_id
uint8 id
